src/azul/service/__init__.py

```diff
@@ -0,0 +1,28 @@
+import ast
+import logging
+
+from azul.service.responseobjects.elastic_request_builder import BadArgumentException
+
+logger = logging.getLogger(__name__)
+
+
+class AbstractService:
+
+    def parse_filters(self, filters: str):
```

Comment:
```suggestion
    def parse_filters(self, filters: Optional[str] = None):
```

Arguments that can be None should have a default value of None. Using type hints and defaults in the signature is much more expressive than prose in the docstring.